src/calculation/ClauseResultsBuilder.ts

```diff
@@ -609,6 +609,15 @@ export function setFinalResults(params: {
     finalResult = FinalResult.NA;
   } else if (params.statementRelevance == Relevance.FALSE || params.rawClauseResults[params.libraryName] == null) {
     finalResult = FinalResult.UNHIT;
+  } else if (
+    params.clause.isFalsyLiteral &&
+    Object.hasOwn(params.rawClauseResults[params.libraryName], params.clause.localId)
```

Comment:
I noticed that this branch is not covered by unit tests... I wonder if there's a good way to test this with the unit tests? Looks like `setFinalResults` does not have any isolated unit tests, but `buildStatementAndClauseResults` (the only function that calls `setFinalResults`) does. Do you think it's possible to add a case to the existing `buildStatementAndClauseResults` test that would fit this branch? Would probably require checking that the `results` have an element where the final result is "true" even though the clause is Null/Literal false.

Not a necessity for merging this since the coverage is not heavily affected by the changes made here - just being mindful of testing in case we need to add more branches in the future